source/faucet/main.d

```diff
@@ -113,7 +113,7 @@ private auto splitTx (UR) (UR utxo_rng, uint count)
                   .drop(uniform(0, 1378 - count, rndGen))
                   .take(count)
                   .map!(k => k.address))
-              .sign()
+              .sign(TxType.Payment, null)
```

Comment:
Both of them are default parameters. You don't need to pass anything here.

Reply:
Fixed